doc/release/1.13.0-notes.rst

```diff
@@ -53,3 +53,10 @@ Improvements
 
 Changes
 =======
+
+``average`` now preserves subclasses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For ndarray subclasses, ``numpy.average`` will now return an instance of the
+subclass, matching the behavior of most other numpy functions such as ``mean``.
+As a consequence, also calls that returned a scalar may now return a subclass
+array scalar.
```

Comment:
Could you also add something under `Compatibility notes`? If you look In the 1.12 notes there is a subsection `FutureWarning to changed behavior` that has a list of changes.